src/popup/index.js

```diff
@@ -70,48 +56,51 @@ export async function focusElement(element, retries = 20, delay = 50) {
     return focusElement(element, retries - 1, delay);
 }
 
-initEmojiMartStorage();
-createPicker().then(async () => {
-    // to be sure, trigger focus manually afterwards
-    // auto-focus does not always work properly, see
-    // https://github.com/rugk/awesome-emoji-picker/issues/28 (now fixed)
-    // https://github.com/rugk/awesome-emoji-picker/issues/86 / https://bugzilla.mozilla.org/show_bug.cgi?id=1623875
-    focusElement(document.querySelector(".emoji-mart-search > input"));
+/**
+ * Resizes the element so it does not overflow anymore or alternatively moves it down.
+ *
+ * Only works on {@see EmojiPicker}/emoji-mart.
+ *
+ * @param {HTMLElement} emojiMartComponent
+ * @param {Symbol} popupType
+ */
+function centerOrResizeDependingOnOverflowOrUnderflow(emojiMartComponent, popupType) {
+    console.info("Detected popup type", popupType, ", centering picker.");
+    emojiMartComponent.style.removeProperty("border");
+    document.body.classList.add("center-picker");
+
+    // setAttribute with boolean flags does not work
+    // see https://github.com/missive/emoji-mart/issues/992
+    emojiMartComponent.setAttribute("dynamicWidth", "true");
```

Comment:
Does this not work?
```suggestion
    emojiMartComponent.dynamicWidth = true;
```

Reply:
Does not seem to work:
![image](https://github.com/user-attachments/assets/0e7f1107-605d-4a13-ba61-907475151dfd)

Also, see the linked issue/workaround below, you can only access it via `.props`.

Reply:
See below/the issue link, apparently `emojiMartComponent.props.dynamicWidth` works. :upside_down_face:

Reply:
Interesting, so maybe this line should be commented out or removed if it does nothing.

Reply:
No the current way works, IIRC.